### PR TITLE
a11y: Add tooltip to icon buttons

### LINKED ISSUE #2006: [Accessibility][ Sensory Characteristics - Flow Chart]: Tooltips are not defined for ‘+’ and other controls.
**User Experience** 
Users having hearing impairment will get difficulties to know the use of the given controls as no text Sensory Characteristics is given for the controls present in calendar.

**Test Environment**
Browser: Microsoft Edge Canary Version: 82.0.425.0 (Official build)
OS build: 2004 (19564.1005)
Screen Reader: Narrator
URL: [Bot Framework Composer](http://localhost:3000/dialogs/Main)
[**Prerequisite Here**](https://github.com/microsoft/BotFramework-Composer)

**Actual Result**
Tooltip of ‘+’ control is not defined.

**Expected:**
Tooltip’s should be defined for the controls which doesn’t have text associated with them.

**Repro Steps:**
1. Open the above mentioned URL in EDGE.
2. Select 'Design Flow' from left navigation.
3. Now, navigate to '+' button with tab in the flow chart.
4. Observe the issue.

**Note:**
Same issue observed throughout the website, tooltips are not defined for any of the controls.

**MAS Impacted:** [MAS1.3.3]( https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={1823da5a-e8fb-4e63-b881-7eccb312c358})

**Attachment:**
[tooltips are not defined for the + button.zip](https://github.com/microsoft/BotFramework-Composer/files/4225289/tooltips.are.not.defined.for.the.%2B.button.zip)

### PR DESCRIPTION
**Pull request by corinagum**
## Description
Add tooltip to IconButtons

## Task Item

Fixes #2006

## Screenshots
tooltip

### CODE REVIEW

**Comment by cwhitten** [comment]
this is nice :)

File: Composer/packages/client/src/App.tsx [hunk 1]
```diff
@@ -148,20 +151,23 @@ export const App: React.FC = () => {
...
+                  setSideBarExpand(!sideBarExpand);
+                }}
+                data-testid={'LeftNavButton'}
+                ariaLabel={sideBarExpand ? formatMessage('Collapse Nav') : formatMessage('Expand Nav')}
+              />
+            </TooltipHost>
             <div css={dividerTop} />{' '}
             <FocusZone allowFocusRoot={true}>
               {topLinks(projectId, openedDialogId).map((link, index) => {
+                const showTooltip = !sideBarExpand && !link.disabled ? true : false;
```
Comment (beyackle) [inline Composer/packages/client/src/App.tsx:170]: See below comment

File: Composer/packages/client/src/App.tsx [hunk 2]
```diff
@@ -148,20 +151,23 @@ export const App: React.FC = () => {
...
+              <IconButton
+                iconProps={{
+                  iconName: 'GlobalNavButton',
+                }}
+                css={globalNav}
+                onClick={() => {
+                  setSideBarExpand(!sideBarExpand);
+                }}
+                data-testid={'LeftNavButton'}
+                ariaLabel={sideBarExpand ? formatMessage('Collapse Nav') : formatMessage('Expand Nav')}
```
[the remark this answers is not in the document]
Comment (corinagum) [reply Composer/packages/client/src/App.tsx:164]: haha woops.

**Comment by corinagum** [comment]
Didn't get to this in time today. I'll do my best to get to it next week.

**Comment by beyackle** [comment]
I'm going to take this up again and see if I can smooth out all the conflicts.

**Comment by beyackle** [comment]
The work here has been reapplied to master in #3362. Closing this one in favor of it.

**Pull request closed without merging**